samigo/pom.xml

```diff
@@ -124,6 +130,27 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+
```

Comment:
Since it's an api and all submodules do use it you can put it here but then you should remove it from all those submodules.

lombok is fine.